fastparquet/api.py

```diff
@@ -122,36 +122,35 @@ def row_group_filename(self, rg):
         else:
             return self.fn
 
-
     def read_row_group_file(self, rg, columns, categories, index=None,
-                            assign=None):
+                            assign=None, mr_times=[]):
```

Comment:
It also seems odd to have this one small dtype case have to manifest itself several times at top-level functions.  This seems fine for this one case, but drives down a path towards kwarg explosion (see pandas.read_csv).